src/euphorie/content/browser/choice.py

```diff
@@ -0,0 +1,24 @@
+from ..option import IOption
+from ..utils import DragDropHelper
+from Acquisition import aq_inner
+from plone.dexterity.browser.view import DefaultView
+from plone.memoize.instance import memoize
+
+
+class ChoiceView(DefaultView, DragDropHelper):
+    @property
+    @memoize
+    def my_context(self):
+        return aq_inner(self.context)
+
+    @property
+    def options(self):
+        return [
+            {
+                "id": option.id,
+                "url": option.absolute_url(),
+                "title": option.title,
+            }
+            for option in self.my_context.values()
```

Comment:
What benefit is giving us to decrease the amount of information we have by converting a full object into a `dict`?
It seems to me that returning the `option` instances instead of the dicts would be better.
The advantages would be that you have more information and less code and indirection.

Reply:
Consistency, I guess. It's done the same way with the solutions in the risk view.